src/escalus_users.erl

```diff
@@ -243,8 +243,8 @@ get_user_by_name(Name) ->
 
 -spec create_user(escalus:config(), named_user()) -> any().
 create_user(Config, {_Name, Options}) ->
-    ClientProps = get_options(Config, Options),
-    {ok, Conn, ClientProps, _} = escalus_connection:start(ClientProps,
+    ClientProps0 = get_options(Config, Options),
+    {ok, Conn, ClientProps, _} = escalus_connection:start(ClientProps0,
```

Comment:
Won't fix. This is intentional - the line in question is a single function call. The equivalence of passed in arguments is coincidental.